ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template5.kt

```diff
@@ -0,0 +1,453 @@
+        ) {
+            Spacer(modifier = Modifier.height(UIConstant.defaultVerticalSpacing))
+        }
+
+        PurchaseButton(state, viewModel)
+
+        Footer(
+            templateConfiguration = state.templateConfiguration,
+            viewModel = viewModel,
+            allPlansTapped = { packageSelectorVisible = !packageSelectorVisible },
+        )
+    }
+}
+
+@Suppress("LongMethod")
+@Composable
+private fun ColumnScope.Template5MainContent(
+    state: PaywallState.Loaded,
+    viewModel: PaywallViewModel,
+    packageSelectionVisible: Boolean,
+) {
+    Column(
+        modifier = Modifier
+            .verticalScroll(rememberScrollState())
+            .conditional(state.isInFullScreenMode) {
+                Modifier.weight(1f)
+            }
+            .padding(horizontal = UIConstant.defaultHorizontalPadding, vertical = UIConstant.defaultVerticalSpacing),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.spacedBy(UIConstant.defaultVerticalSpacing, Alignment.CenterVertically),
+    ) {
+        if (state.isInFullScreenMode) {
+//            Spacer(Modifier.weight(1f))
+
+            val localizedConfig = state.selectedLocalization
+            val colors = state.templateConfiguration.getCurrentColors()
+            Markdown(
+                style = MaterialTheme.typography.displaySmall,
+                fontWeight = FontWeight.Bold,
+                textAlign = TextAlign.Left,
+                text = localizedConfig.title,
+                color = colors.text1,
+            )
+            Spacer(Modifier.weight(1f))
+
+            Features(
+                features = localizedConfig.features,
+                colors = colors,
+            )
+
+            Spacer(Modifier.weight(1f))
+        }
+
+        AnimatedPackages(state, packageSelectionVisible, viewModel)
+
+        if (state.isInFullScreenMode) {
+            Spacer(Modifier.weight(1f))
+        }
+    }
+}
+
+@Composable
+private fun HeaderImage(uri: Uri?) {
+    uri?.let {
+        RemoteImage(
+            urlString = uri.toString(),
+            modifier = Modifier
+                .aspectRatio(ratio = Template5UIConstants.headerAspectRatio),
+            contentScale = ContentScale.Crop,
+        )
+    }
+}
+
+@Composable
+private fun Features(
+    features: List<PaywallData.LocalizedConfiguration.Feature>,
+    colors: TemplateConfiguration.Colors,
+) {
+    features.forEach { feature ->
+        Feature(
+            feature = feature,
+            colors = colors,
+        )
+    }
+}
+
+@Composable
+private fun Feature(
+    feature: PaywallData.LocalizedConfiguration.Feature,
+    colors: TemplateConfiguration.Colors,
+) {
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = Modifier
+            .fillMaxWidth()
+            .semantics(mergeDescendants = true) {},
+    ) {
+        feature.iconID?.let { PaywallIconName.fromValue(it) }?.let { icon ->
```

Comment:
I see that if the icon is not given or it's unknown for a specific feature, the text will use the full row, and not be aligned with the others. Should we default to a specific icon or at least leave an empty space so in case we add new icons in the future and the SDK doesn't know the specific icon, the text is at least aligned?

Reply:
Great catch! Fixed this by moving the Box out:
<img width="408" alt="Screenshot 2023-10-30 at 10 32 04" src="https://github.com/RevenueCat/purchases-android/assets/685609/b783fd7e-6769-4d8d-9a11-fd6b966d19ab">